integration/report_definitions_update_test.go

```diff
@@ -0,0 +1,121 @@
+//go:build report_definitions && !windows
+
+// Author:: Darren Murray (<dmurray-lacework@lacework.net>)
+// Copyright:: Copyright 2023, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package integration
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/Netflix/go-expect"
+	"github.com/lacework/go-sdk/api"
+	"github.com/lacework/go-sdk/cli/cmd"
+	"github.com/stretchr/testify/assert"
+	"gopkg.in/yaml.v2"
+)
+
+func TestReportDefinitionsPromptUpdate(t *testing.T) {
+	report := fetchCustomReportDefinition()
```

Comment:
What if someone deletes the report definition `CLI Test`? also, how do we pre-provision this resource in other Lacework accounts like QAN, are you planning to add some Terraform?

Could we maybe create a new test report definition instead of fetching it from the account?

Reply:
👍